runtime/bcutil/j9bcu.tdf

```diff
@@ -373,4 +373,4 @@ TraceException=Trc_BCU_j9bcutil_readClassFileBytes_MaxCPCount NoEnv Overhead=1 L
 
 TraceException=Trc_BCU_internalDefineClass_orphanNotFound Overhead=1 Level=1 Template="orphan ROM class %p not found in table, table entry was %p"
 
-TraceEvent=Trc_BCU_compareROMClassForEquality_event NoEnv Overhead=1 Level=7 Template="compareROMClassForEquality returns %d for class name %.*s"
+TraceEvent=Trc_BCU_compareROMClassForEquality_event NoEnv Overhead=1 Level=7 Template="For class name %.*s, compareROMClassForEquality returns %d"
```

Comment:
We're not supposed to change tracepoint parameters - not sure if this matters any more. This change seems unnecessary anyway, as it doesn't provide any clarification.

Reply:
@DanHeidinga

Reply:
If the order of tracepoint parameters is changed, this line should be updated too:
https://github.com/eclipse/openj9/blob/master/runtime/bcutil/ROMClassBuilder.cpp#L1341

Reply:
Tracepoint text is not generally visible (and it's not translated), so I'm strongly in favour of not making this part of the change.

Reply:
I agree with GAC, better to leave the tracepoint alone in this case.

Reply:
I changed the tracepoint because in the SCCMLTest I was checking if
```
compareROMClassForEquality returns 1 for class name org/openj9/test/lambdatests/Test1..Lambda.([\d]+)/0000000000000000
```
existed but I needed to change it to
```
compareROMClassForEquality returns 1 for class name org/openj9/test/lambdatests/Test1..Lambda.([\d]+)/(00000000|0000000000000000)
```
 for 32 bit machines. But I realized this would also match anything starting with
```
org/openj9/test/lambdatests/Test1$$Lambda$1/00000000
```
as long as there are 8 0's at the beginning, for example:
```
org/openj9/test/lambdatests/Test1$$Lambda$1/0000000012345678
```
But with this change, the ',' after the class name would prevent this from happening. I could also do this with adding any character at the end of the previous tracepoint if that is better.

Reply:
Changing the text of the tracepoint is not an issue, as long as the parameter order remains the same.

Reply:
I added a space character at the end of the previous tracepoint and checked for that. Could you please take a look?

Reply:
I think that's a bit too obscure. Can you quote the name instead?

`TraceEvent=Trc_BCU_compareROMClassForEquality_event NoEnv Overhead=1 Level=7 Template="compareROMClassForEquality returns %d for class name '%.*s'"`

Reply:
I added the quotes. Could you please review again?

Reply:
Much better, thanks.